com.unity.render-pipelines.universal/Runtime/Debug/DebugHandler.cs

```diff
@@ -0,0 +1,337 @@
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Diagnostics;
+using UnityEditor.Rendering;
+
+namespace UnityEngine.Rendering.Universal
+{
+    public class DebugHandler : IDebugDisplaySettingsQuery
```

Comment:
Does this class need to be public?

Reply:
I think it could be made internal. Seems like I missed this I did a pass on changing public to internal where possible, I'll revisit this one.

Reply:
Made internal